xml/Microsoft.Extensions.Hosting.Internal/ApplicationLifetime.xml

```diff
@@ -86,10 +96,19 @@
         <ReturnType>System.Threading.CancellationToken</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Triggered when the application host has fully started and is about to wait
-            for a graceful shutdown.</summary>
+        <summary>Gets a cancellation token. Triggered when the application host has fully started and is about to wait for a graceful shutdown.</summary>
```

Comment:
@maryamariyan Thanks for reviewing. I didn't understand these properties, so can you double check the modifications I made? The descriptions make them sound more like events. I'm not sure how a property is "triggered".

Reply:
Should be fine I guess. I also found similar wording here: https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.iapplicationlifetime?view=dotnet-plat-ext-5.0